src/frontend/Dockerfile

```
@@ -22,9 +22,12 @@ FROM frontend-deps AS drive-dev
 
 WORKDIR /home/frontend/apps/drive
 
+# Add local node_modules/.bin to PATH
+ENV PATH="/home/frontend/apps/drive/node_modules/.bin:$PATH"
+
 EXPOSE 3000
 
-CMD [ "yarn", "dev"]
+RUN yarn && yarn build-theme && yarn dev
```

```suggestion
RUN yarn build-theme 
CMD ["yarn", "dev"]
```
The build will never end as this step will be blocked.
Changing the `CMD` instruction by `RUN` remove the possibility to change the command and skip the entrypoint usage.
Running yarn again should be also removed, this is done in the `frontend-deps` stage

---

> Running yarn again should be also removed, this is done in the frontend-deps stage

yes, sorry, this was for debugging purposes and I left it there